### ISSUE TITLE
Safari combination of Layout Horizontal Wrap and child with Min-Width broken

### ISSUE THREAD

**Issue by mbergeron** [PolymerElements/iron-flex-layout#35]
Hi,

I don't know if the issue comes from the iron-flex-layout or Safari's render engine.

The followings works on Chrome, Firefox, IE and Edge. Only Safari doesn't wrap the child elements

This is the code basics

CSS

        .testContainer{
            background-color:lightGray;
            padding:5px;
        }
        
        .testBox{
            background-color:whitesmoke;
            margin:5px;
            min-width: 500px;
        }

HTML

           <div class="layout horizontal wrap testContainer">
                <div class="flex testBox">min-width:500px
                </div>
                <div class="flex testBox">min-width:500px
                </div>
                <div class="flex testBox">min-width:500px
                </div>
            </div>

![image](https://cloud.githubusercontent.com/assets/1834714/10193807/2f1a8d1e-6753-11e5-8dfb-03b1ac8a7dec.png)

**Comment by notwaldorf** [comment]
Eeek, sorry for the delay in answering this. First, here's a jsbin reproducing it, with both native flex css, and `iron-flex-layout`: http://output.jsbin.com/zelusabayo

It seems that the same thing happens with plain, non-polymer flexbox because of a Safari bug: http://stackoverflow.com/questions/25360526/row-wrap-in-flex-box-not-wrapping-in-safari/30792851#30792851

The solution is to add `-webkit-flex: 1 1 500px;` to your flex child, and that will fix it. I've added this to the demo.

Closing this, as it's not an `iron-flex-layout` bug.
